challenges/objects-arrays.js

```diff
@@ -80,8 +116,15 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 let lowerCase = [];
+zooAnimals.map(function(value){
```

Comment:
Consider `animalName` in lieu of `value`. 
You may want to use arrow functions for map and reduce methods: 

`let lowerCase = zooAnimals.map((animalName) => {
 return animalName.animal_name.toLowerCase();
});
console.log(lowerCase); `